zospy/zpcore.py

```diff
@@ -318,6 +318,8 @@ def __init__(self):
 
         logger.info("ZOS instance initialized")
 
+        self.wakeup()
```

Comment:
Please make sure the arguments for wakeup are parsed. 

```suggestion
        self.wakeup(preload=preload, zosapi_nethelper=zoapi_nethelper)
```

To do so, they have to be made available in __init__. I think zosapi_nethelper requires to be an argument, as the possibility of passing the path to this nethelper is relevant. For preload, I think this is not required for most users and might be added as a kwarg. @crnh do you agree? If so, the code above can be changed to `preload=kwargs.get(preload, False)`

Reply:
I personally do not really see the advantage of using `**kwargs` here as we don't deal with a large number of parameters, and `**kwargs` tends to make the interface more opaque.
I propose to add them both in the same way as is currently done in `ZOS.wakeup`.